src/main/java/org/alfresco/repo/search/impl/querymodel/impl/db/AspectSupport.java

```diff
@@ -100,20 +105,41 @@ public void buildJoins(Map<QName, DBQueryBuilderJoinCommand> singleJoins, List<D
      */
     @Override
     public void buildPredicateCommands(List<DBQueryBuilderPredicatePartCommand> predicatePartCommands)
-    {
-        DBQueryBuilderPredicatePartCommand command = new DBQueryBuilderPredicatePartCommand();
-        command.setJoinCommandType(DBQueryBuilderJoinCommandType.ASPECT);
-        command.setAlias(alias);
-        command.setType(DBQueryBuilderPredicatePartCommandType.ASPECT);
-        if(qnameIds.size() > 0)
-        {
-            command.setValues(qnameIds.toArray(new Long[]{}));
-        }
-        else
-        {
-            command.setValues(new Long[]{-1l});
-        }
-        predicatePartCommands.add(command);
-    }
+    {
+    	DBQueryBuilderPredicatePartCommand command;
+    	switch(joinType)
+    	{
+    	case NONE:
```

Comment:
Formatting

Reply:
this was fixed in 9142625fd883c184d9850e52cf6254d3360b5e91   ??